### ISSUE TITLE
1.21 Update

### ISSUE THREAD

**Issue by dylanpdx** [Retro64Mod/Retro64Mod.github.io#200]
Things needed for a functional 1.21 update:
- [x] Port to NeoForge
- [x] Have it actually able to start
- [x] Have Mario mode working
- [x] Port over capabilities to data attachments
- [x] Multiplayer rewrite
- [x] Bug testing / Fixing
- [x] ??? More things may crop up

**Comment by JavaJacket1** [comment]
I noticed You said " things needed for 1.21" I know You might have implied it but does this mean You are working on a port?(sorry if I sound rude)

And if so do You plan to make a 1.20 port as well? Or just 1.21? Either way I thank You for sinking some time back into this project!

**Comment by dylanpdx** [comment]
Yep, I'm finally dedicating some time to properly updating the mod to 1.21. Mainly just a ton of changes to sift through in terms of how MC has changed
Not sure about 1.20, but assuming the 1.21 update goes well I could try my hand at a backport

**Comment by JavaJacket1** [comment]
I love it man thank You so much for Your hard work, I really do appreciate it!

**Comment by FatalERROR1234** [comment]
> Yep, I'm finally dedicating some time to properly updating the mod to 1.21. Mainly just a ton of changes to sift through in terms of how MC has changed Not sure about 1.20, but assuming the 1.21 update goes well I could try my hand at a backport

How long till the release date (or a rough estimate)?
